database_manager.py

```diff
@@ -7,14 +7,12 @@ def __init__(self, database_connector):
     
     def add_item(self):
         item_id = database_helper.get_uuid()
-        self.database_connector.execute("INSERT INTO items (item_id, vector_path, png_path) VALUES (%s, %s, %s)", (item_id, item_id+"_vector", item_id+"_png"))
+        self.database_connector.execute("INSERT INTO Items (ItemId, VectorPath, PngPath) VALUES (%s, %s, %s)", (item_id, item_id+"_vector", item_id+"_png"))
```

Comment:
Why add _vector and _png?

Reply:
currently added as identifiers for each item, will likely just be item_id with file ending later but leaving like this to be able to identify the data more easily in the FE until the actual file references can be associated correctly. Should I do it some other way perhaps?